Homework_7/src/ParserPackage/FileParser.java

```diff
@@ -0,0 +1,108 @@
+package ParserPackage;
+
+import java.io.*;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class FileParser {
+    private File fileForReading ;
+    public File containsFile(String fileName){
+        File bookFile = new File("src\\"+fileName);
+        if (!bookFile.exists()) {
+           throw new RuntimeException("Book is not found !");
+        }
+        fileForReading=bookFile;
+        return bookFile;
+    }
+    public  List<String> clearningStrings (List<String> inputStringSequence){
+        Pattern regularExpression =Pattern.compile("((^\\W*)([a-zA-Z]+\\'?\\-?[a-zA-Z]*)(\\W*$))");
```

Comment:
^\\W*)([a-zA-Z]+\\'?\\-?[a-zA-Z]*)(\\W*$))" can be class level static string  and var name is not good